Update: Add supported versions to Moesif plugin. Add UTM Params (#2240)
Kong/docs.konghq.com: app/_hub/moesif/kong-plugin-moesif/index.md

---
name: Moesif API Analytics
publisher: Moesif

categories: 
  - analytics-monitoring

>   - logging

Reviewer: @lena-larionova are there any (filter or other) issues with having multiple category selections?

Reviewer 2: No technical issues, but we've had bug reports filed for plugins that sit in two categories - people get confused because the plugin displays twice on the page. Ideally, we don't want to use two categories, and instead pick the one that applies the most.

type: plugin

desc: User-centric API analytics and logging  

description: |

>   Monitor API traffic and analyze customer usage in [Moesif's API Analytics](https://www.moesif.com/solutions/track-api-program?language=kong-api-gateway&utm_medium=docs&utm_campaign=partners&utm_source=kong) platform, enabling:

Reviewer: "enabling" on its own isn't a grammatically correct lead-in to the list items below. Suggest:

```suggestion
  Monitor API traffic and analyze customer usage in [Moesif's API Analytics](https://www.moesif.com/solutions/track-api-program?language=kong-api-gateway&utm_medium=docs&utm_campaign=partners&utm_source=kong) platform, which enables you to:
```

The lead-in should work as a sentence with each list item.

  * [Understand customer API usage](https://www.moesif.com/features/api-analytics?utm_medium=docs&utm_campaign=partners&utm_source=kong) with user behavior analytics.
  * [Debug issues quickly](https://www.moesif.com/features/api-logs?utm_medium=docs&utm_campaign=partners&utm_source=kong) with high-cardinality API logs and metrics.

>   * [Get alerted](https://www.moesif.com/features/api-monitoring?utm_medium=docs&utm_campaign=partners&utm_source=kong) of problems impacting customers.

Reviewer: ```suggestion
  * [Get alerted](https://www.moesif.com/features/api-monitoring?utm_medium=docs&utm_campaign=partners&utm_source=kong) of problems that impact customers.
```

make tense // and consistent with the entry "that keep customers informed"

    #incompatible:
  enterprise_edition: # optional
    compatible:
      - 2.1.x-x
      - 1.5.x-x

>       - 1.3.x-x

Reviewer: ```suggestion
      - 1.3-x-x
```

@lena-larionova is the above extra x how we should be formatting versions for KE and CE? I have my plugin version compatibility PR in progress.

Reviewer: I am not seeing these show up in the side panel 

![image](https://user-images.githubusercontent.com/3756245/90649615-8b1c8180-e200-11ea-902b-d9c81efc127a.png)

Reviewer 2: > @lena-larionova is the above extra x how we should be formatting versions for KE and CE? I have my plugin version compatibility PR in progress.

No - the formatting of these versions should match what's used in the site URLs, or the doc site version dropdown. In this case, the versions should be 1.3-x, 1.5.x, and 2.1.x

### How it works

> When enabled, this plugin will capture API traffic and log to

Reviewer: ```suggestion
When enabled, this plugin captures API traffic and logs it to
```

[Moesif API Analytics](https://www.moesif.com/?language=kong-api-gateway&utm_medium=docs&utm_campaign=partners&utm_source=kong). 

> This plugin adds zero latency and does not sit in line to API traffic.

Reviewer: what does "does not sit in line to API traffic" mean? It doesn't sit in a queue? It doesn't interfere with API traffic? Both?
